### PR TITLE
Pin node version at 14.19.3

### PR DESCRIPTION
**Pull request by jacksonj04**
This explicitly pins the node version for people using version managers, as well as changing the Github action to install the pinned version.

### CODE REVIEW

**Review by pezholio — APPROVED** [review]
LGTM - though there's a conflict that needs resolving

File: .nvmrc [hunk 1]
```diff
@@ -0,0 +1 @@
+v14.19.3
```
Comment (Gweaton) [inline .nvmrc:1]: Is `nvm` the thing most people use? I've been using `nodenv`, as it's essentially the same as `rbenv` but I don't know what the majority of people at dxw are using!

Comment (pezholio) [reply .nvmrc:1]: That's a good point - I used `nodenv` too - didn't clock the filename!

Comment (jacksonj04) [reply .nvmrc:1]: Swapped to using `.node-version` instead - `nvm` looks to be heading for support for this anyway.

**Pull request merged**